schema/rdl_schema_0.1.json

```
@@ -97,15 +100,29 @@
       "$ref": "#/$defs/common_license",
       "description": "Type of license"
     },
-    "maintainer": {
-      "type": "string",
-      "description": "Who added the resource",
-      "minLength": 1
+    "contact_point": {
+      "title": "Contact point",
+      "type": "object",
+      "description": "Contact information for the maintainer of the dataset.",
+      "$ref": "#/$defs/Entity",
+      "minProperties": 2
```

`minProperties` is already set on `Entity`.

```suggestion
      "$ref": "#/$defs/Entity"
```